src/index.html

```diff
@@ -7,8 +7,36 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <nav class="box-logo">
+        <a href="#home">
+          <img
+            src="images/logo.png"
+            alt="logo"
+            class="logo"
+          >
+        </a>
+      </nav>
+      <nav class="box-title">
+        <ul class="link-box">
+          <a href="#home" class="link is-active hover">Apple</a>
```

Comment:
```suggestion
          <li>
            <a href="#home" class="link is-active hover">Apple</a>
          </li>
```
links should be inside of list items

Reply:
done